Makefile

```diff
@@ -264,7 +264,7 @@ ut: cluster-create run-uts cluster-destroy
 run-uts:
 	-mkdir -p .go-pkg-cache report
 	$(CONTAINERIZED) sh -c '$(GIT_CONFIG_SSH) \
-	ginkgo -r --skipPackage -focus="$(GINKGO_FOCUS)" $(GINKGO_ARGS) $(WHAT)'
+	ginkgo -r --skipPackage ./external -focus="$(GINKGO_FOCUS)" $(GINKGO_ARGS) $(WHAT)'
```

Comment:
This is exactly the kind of change that results in us accidentally skipping all tests 😅 

I verified locally that running this still executes tests. Though I do wonder why we passed an empty `--skipPackage` in the first place... were we skipping all tests and then `-focus` all or something?